crates/libzkp/src/tasks/chunk.rs

```diff
@@ -1,16 +1,20 @@
-use super::chunk_interpreter::*;
 use eyre::Result;
 use sbv_core::BlockWitness;
-use sbv_primitives::B256;
+use sbv_primitives::{types::consensus::BlockHeader, B256};
 use scroll_zkvm_types::{
-    chunk::{execute, ChunkInfo, ChunkWitness, LegacyChunkWitness},
+    chunk::{execute, ChunkInfo, ChunkWitness, LegacyChunkWitness, ValidiumInputs},
     task::ProvingTask,
     utils::{to_rkyv_bytes, RancorError},
+    version::Version,
 };
 
+use super::chunk_interpreter::*;
+
 /// The type aligned with coordinator's defination
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 pub struct ChunkTask {
+    /// The version for the chunk, as per [`Version`].
+    pub version: u8,
```

Comment:
Maybe we can also put the "decryption_key" field also into `ChunkTask` so saving a argunment in calling?